onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/tensor/resize.h"
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+
+namespace onnxruntime {
+namespace test {
+
+TEST(ResizeOpTest, ResizeOpNearestTest) {
+  OpTester test("Resize", 10);
+  std::vector<float> scales{1.0f, 1.0f, 0.6f, 0.6f};
```

Comment:
maybe we should consider the case when scales is larger than 1?

Reply:
that case was covered by previous upsample test cases.